docs/apache-airflow-providers-alibabacloud/operators/oss.rst

```diff
@@ -0,0 +1,64 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+Alibaba OSS Operators
+====================
+
+.. contents::
+  :depth: 1
+  :local:
+
+Overview
+--------
+
+Airflow to Alibaba Cloud Object Storage Service (OSS) integration provides several operators to create and interact with OSS buckets.
+
+ - :class:`~airflow.providers.alibabacloud.sensors.oss_key.OSSKeySensor`
+ - :class:`~airflow.providers.alibabacloud.operators.oss_bucket.OSSCreateBucketOperator`
+ - :class:`~airflow.providers.alibabacloud.operators.oss_bucket.OSSDeleteBucketOperator`
+ - :class:`~airflow.providers.alibabacloud.operators.oss_object.OSSUploadObjectOperator`
+ - :class:`~airflow.providers.alibabacloud.operators.oss_object.OSSDownloadObjectOperator`
+ - :class:`~airflow.providers.alibabacloud.operators.oss_object.OSSDeleteBatchObjectOperator`
+ - :class:`~airflow.providers.alibabacloud.operators.oss_object.OSSDeleteObjectOperator`
+
+Two example_dags are provided which showcase these operators in action.
+
+ - example_oss_bucket.py
+ - example_oss_object.py
+
+.. _howto/operator:OSSCreateBucketOperator:
+.. _howto/operator:OSSDeleteBucketOperator:
+
+Create and Delete Alibaba Cloud OSS Buckets
+-----------------------------------
+
+Purpose
+"""""""
+
+This example dag ``example_oss_bucket.py`` uses ``OSSCreateBucketOperator`` and ``OSSDeleteBucketOperator`` to create a
```

Comment:
Please remove reference to example dag in the description and focus only on the description of operator usage.